app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/ConfigurationPage.vue

```diff
@@ -103,6 +103,30 @@
       >
         <woot-code :script="inbox.provider_config.api_key" />
       </settings-section>
+      <settings-section
+        :title="$t('INBOX_MGMT.SETTINGS_POPUP.WHATSAPP_SECTION_UPDATE_TITLE')"
+        :sub-title="$t('INBOX_MGMT.SETTINGS_POPUP.WHATSAPP_SECTION_SUBHEADER')"
```

Comment:
We can add a new subheader, with a copy like this:

"Enter the updated key to be used for the integration with the WhatsApp APIs."